SU-SWS/ace-sdssgryphon
# SDSS-479: setting the default card type

# READY FOR REVIEW

# Summary
- Set the default card type to Topic

# Review By (Date)
- 2/16

# Urgency
- Normal

# Steps to Test

1. Checkout this branch.
2. Use the sustainability db so you can see this page /qa-card-styles
3. Compare your local to live to see no changes.
4. Add a card and look a the styles tab to verify there is now a default to Topic for type.
5. Make a coup of cards and see how they look.

# Affected Projects or Products
- SDSS Subtheme

# Associated Issues and/or People
[- SDSS-479](https://stanfordits.atlassian.net/browse/SDSS-479)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
